docs/avatar.html

```diff
@@ -5,24 +5,120 @@
-      </div>
-      <div class="badge badge-wrapper">
-        <div class="far fa-bell"></div>
-        <div class="badge-counter badge-top-left badge-clr-red badge-round">8</div>
-      </div>
-      <div class="badge badge-wrapper">
-        <div class="far fa-heart"></div>
-        <div class="badge-counter badge-top-right badge-clr-green badge-round">9+</div>
-      </div>
-      <div class="badge badge-wrapper">
-        <div class="fas fa-shopping-cart"></div>
-        <div class="badge-counter badge-top-left badge-clr-orange badge-round">2</div>
-      </div>                              
+    <nav class="navigation">
+        <h1 class="logo">parts</h1>
+        <div class="nav-links">
+            <span>Docs</span><span>Github</span>
+        </div>
+    </nav>
+    <div class="sidebar-and-content">
+        <div class="sidebar">
+            <h2>Components</h2>
+            <ul>
+                <li><a href="avatar.html">Avatar</a></li>
+                <li><a href="alert.html">Alert</a></li>
+                <li><a href="badge.html">Badge</a></li>
+                <li><a href="button.html">Button</a></li>
+                <li><a href="card.html">Card</a></li>
+                <li><a href="image.html">Image</a></li>
+                <li><a href="input.html">Input</a></li>
+                <li><a href="text-utilities.html">Text Utilities</a></li>
+                <li><a href="lists.html">Lists</a></li>
+                <li><a href="navigation.html">Navigation</a></li>
+                <li><a href="modal.html">Modal</a></li>
+                <li><a href="rating.html">Rating</a></li>
+                <li><a href="toast.html">Toast/Notify/Snackbar</a></li>
+                <li><a href="simplified-grid.html">Simplified Grid</a></li>
+                <li><a href="navigation.html">Navigation</a></li>
+                <li><a href="slider.html">Slider</a></li>
+            </ul>
+        </div>
+        <div class="content">
+            <div class="description">
+                <h3>Avatar</h3>
+                <h4>The Avatar is used to represent a user, and displays the profile picture. Here, we have 3 avatar components:</h4>
+                <h4>1. Basic avatar icon</h4>
+                <h4>2. Fallback: In case user pic is not available</h4>
+                <h4>3. Avatars with different sizes</h4>
+            </div>
+    
+            <div>
+            <p>1. Basic avatar icon:</p>
+            <div class="avatar-container basic">
+                <img class="avatar avatar-one" src="/pics/avatar/avatar1.jpg" alt="">
+                <img class="avatar avatar-two" src="/pics/avatar/avatar2.jpg" alt="">
+                <img class="avatar avatar-three" src="/pics/avatar/avatar3.jpg" alt="">
+                <img class="avatar avatar-four" src="/pics/avatar/avatar4.jpg" alt="">
+                <img class="avatar avatar-five" src="/pics/avatar/avatar5.jpg" alt="">
```

Comment:
Add values to `.alt`

Reply:
Devananth, I am not sure but maybe you have to resolve the conservation.

Reply:
I think `.alt` is important , it helps in **accessiblity**. I would recommend you to add it.

Reply:
The required alt attribute specifies an alternate text for an image if the image cannot be displayed. The alt attribute provides alternative information for an image if a user for some reason cannot view it. 
As of now I can't think of any alternative to use in place of those images.
I'll keep it in mind if I observe that those images are slow to load. 

Thanks for letting me know about alt (I did not knew its purpose before)

Reply:
Okay , Proceed !

Reply:
👍